jax_distribution/types.py

```diff
@@ -42,8 +43,22 @@ class RunnerState(NamedTuple):
     timestep: TimeStep
 
 
+class EvalState(NamedTuple):
```

Comment:
Sorry for all the renames, thinking that if you now have `EvalState` we should maybe rename `RunnerState` to `LearnerState` or maybe `ActorState`, what do you think?

Reply:
yes totally agree we will change it to LearnerState